01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,21 @@
+const http = require("http");
+
+//req represetns incoming requests from the client
+//res is what we're sending back
+const server = http.createServer((req, res) => {
+  //console.log(req);
+  if (req.url === "/") {
+    res.end("Welcome to our home page!");
+  }
+  if (req.url === "/about") {
+    res.end("Here is our short history");
+  }
+  res.end(
+    `<h1>Oops! </h1>
+    <p>We can't seem to find the page you are looking for :(</p>
+    <a href="/">back home</a>`
+  );
```

Comment:
Hi @Nata-Kalina 👋🏼. This is close, but when I make a request against the server it produces the following error:

![image](https://user-images.githubusercontent.com/22882623/212210849-dceb8eba-ea29-4510-9cab-d71fd720bcab.png)

That's because `res.end` can be called more than once with the current code. You can either (1) use `else if` and `else` blocks to ensure it can only be called once or (2) make an 'early return' in each `if` clause to stop processing the request after you send a response.

Reply:
Hi @taylor-knapp! 🤝 Thank you for your review and for your prompts! I fixed the file 12-http.js and added new commit. I don't know what to do next, should I close the previous pull request and open new, or all my 2 commits will be added to existing pull request? Thank you for your time and patience 🙂

Reply:
The PR is based on the associated branch so now that you've added the commit and pushed it's good to go! Nice fix!

Reply:
Thank you!